<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Human_Widget</class>
 <widget class="QWidget" name="Human_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Human_Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>421</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="setagebox">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>年龄设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="seetagelay">
         <item>
          <widget class="QLabel" name="setagetext">
           <property name="text">
            <string>设置年龄</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="setageslider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBothSides</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ageintlay">
         <item>
          <widget class="QLabel" name="agereslabel">
           <property name="text">
            <string>ageChange(int)响应</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="agerestext"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="agestrlay">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>ageChange(str)响应</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="setnamebox">
      <property name="title">
       <string>姓名设置</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>421</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="name">
            <property name="text">
             <string>输入姓名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="text">
             <string>Mike</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>设置姓名</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="namelabel">
            <property name="text">
             <string>nameChange(str)响应</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nametext"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="close">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>281</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/jpgs/images/time.jpg);
color: rgb(255, 0, 0);
font: 30pt &quot;楷体&quot;;
color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>close</zorder>
      <zorder>layoutWidget</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>Human_Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
